.github/workflows/publish_to_dockerhub.yml

```diff
@@ -43,3 +43,30 @@ jobs:
       - name: Push Prod Tags
         if: steps.check-tag.outputs.match == 'true'
         run: nox -s "push(prod)"
+  push-fidesops-privacy-center-image:
```

Comment:
I followed the Github documentation on this, but don't have any way to actually test it.